.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,72 @@
+name: Build and Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
Should this be here?

Reply:
What do you mean ? if we want to have https://nodejs-laoders.github.io we need to deploy to GH pages